database/worker.go

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package database
+
+import (
+	"database/sql"
+
+	"github.com/go-vela/types/library"
+)
+
+// Worker is the database representation of a worker.
+type Worker struct {
+	ID            sql.NullInt64  `sql:"id"`
+	Hostname      sql.NullString `sql:"hostname"`
+	Path          sql.NullString `sql:"path"`
```

Comment:
Did you have any other ideas than `path` to hold the address of a worker? `path` doesn't seem very intuitive to me personally.

Reply:
i'm open to suggestions. was originally url, but i wasn't thrilled about either choice.

Reply:
I would prefer url over path. The only ideas I have would be like address, URL, ingress, location.

Reply:
I agree that I'm not a big fan of `path` since it could cause confusion.

I'm thinking specifically in technology, `path` often refers to an actual path to a file or directory on a host.

`URL` or `Address` would work for me 👍

Reply:
My vote would be for address.

Reply:
I'm okay with address, I'll change to that